listenbrainz/spark/request_manage.py

```diff
@@ -91,8 +90,37 @@ def request_all_user_stats():
     """
     send_request_to_spark_cluster(_prepare_query_message('stats.user.all'))
 
+
 @cli.command(name="request_import_full")
 def request_import_new_full_dump():
     """ Send the cluster a request to import a new full data dump
     """
     send_request_to_spark_cluster(_prepare_query_message('import.dump.full'))
+
+
+@cli.command(name="request_dataframes")
+def request_dataframes():
```

Comment:
we should have tests for each of these commands in `test_request_manage.py` ideally.